README.md

```diff
@@ -3,7 +3,7 @@
   align="center" title="Opal logo by Elia Schito" width="105" height="105" />
   <br/>
   Opal  <br/>
-  <img src="https://img.shields.io/badge/Opal-Ruby%20💛%20JavaScript-yellow.svg?logo=ruby&style=social&logoColor=777"/>
+  <img src="https://img.shields.io/badge/Opal-Ruby%20💛%20JavaScript-yellow.svg?logo=ruby&style=social&logoColor=777" alt="Opal-Ruby💛JavaScript"/>
```

Comment:
🙏 thanks for adding all the alt attributes
nit: can you please add a space before and after the yellow heart?

Reply:
Done 😄👍